core-bundle/tests/Routing/Route404ProviderTest.php

```diff
@@ -68,14 +81,27 @@ public function testGetRoutesByNamesWithoutValueReturnsAllRoutes(): void
 
         $framework = $this->mockContaoFramework([PageModel::class => $pageAdapter]);
 
-        $provider = new Route404Provider($framework, false);
+        $candidates = $this->createMock(Candidates::class);
+//        $candidates
+//            ->expects($this->once())
+//            ->method('getCandidates')
+//            ->willReturn([])
+//        ;
```

Comment:
This should be uncommented or removed.

Reply:
Implemented in 164291bb170c7ec3450f87083c41ed0e03176c99